components/Layout.tsx

```diff
@@ -289,14 +290,30 @@ const Footer = () => (
       <a href='https://github.com/json-schema-org/community/discussions' className='text-sm text-slate-400 hover:text-slate-500 pt-3'>GitHub Community Discussions</a>
       <a href='https://groups.google.com/g/json-schema' className='text-sm text-slate-400 hover:text-slate-500 pt-3'>Google Groups</a>
     </div>
-    <div className='flex flex-col items-start mt-8 sm:mt-16 lg:mt-0'>
+    <div className='flex flex-col items-start mt-8 sm:mt-8 lg:mt-0'>
       <Logo />
       <a href='https://opencollective.com/json-schema' className='text-sm text-slate-400 hover:text-slate-500 pt-3 mt-2'>Open Collective</a>
       <a href='/understanding-json-schema/credits' className='text-sm text-slate-400 hover:text-slate-500 pt-3'>Acknowledgments</a>
     </div>
   </footer>
 )
 
+const OpenJS = () => (
```

Comment:
I reviewed how this 2 columns footer looks like and I'd like to suggest trying single column approach like https://loopback.io/

![Screenshot 2023-04-12 at 10 42 15](https://user-images.githubusercontent.com/40007659/231403017-d3915983-0bba-49f0-ad92-3a4a63a18a9a.png)

Reply:
Pushed a commit revising the responsive layout for OpenJS artifacts and adjusted the overl padding and margins for the footer.